src/lib/flight_tasks/tasks/Orbit/FlightTaskOrbit.cpp

```diff
@@ -224,5 +231,37 @@ void FlightTaskOrbit::generate_circle_setpoints(Vector2f center_to_position)
 	_position_setpoint(0) = _position_setpoint(1) = NAN;
 
 	// yawspeed feed-forward because we know the necessary angular rate
-	_yawspeed_setpoint = _v / _r;
+	_yawspeed_setpoint = NAN; //_v / _r;
+
+	switch (_yaw_behaviour) {
```

Comment:
I was asking to put this logic in its own function. Now it's not in the main update anymore but just uses another already existing one.

Reply:
Is generate_circle_setpoints not the proper place? Or do you prefer an additional generate_circle_yaw_setpoints function?